modules/assisted-installer-how-to-use-the-assisted-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: CONCEPT
+
+[id="how-to-use-the-assisted-installer_{context}"]
+= How to use the Assisted Installer
+
+The {product-title} link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[{ai-full}] is a user-friendly installation solution offered on the link:http://console.redhat.com[Red Hat Hybrid Cloud Console]. The {ai-full} supports the various deployment platforms with a focus on bare metal.
+
+The {ai-full} provides installation functionality as a service. This software-as-a-service (SaaS)  approach has the following advantages:
+
+* *Web user interface:* The web user interface performs cluster installation without the user having to create the installation configuration files manually.
+* *No bootstrap node:* A bootstrap node is not required when installing with the {ai-full}. The bootstrapping process executes on a node within the cluster.
+* *Hosting:* The {ai-full} hosts:
+  - Ignition files
+  - The installation configuration
+  - A discovery ISO
+  - The installer
+* *Simplified deployment:* Deployment does not require in-depth knowledge of {product-title}. The {ai-full} provides reasonable defaults. Additionally, the {ai-full} performs rich pre-flight validation before installation to catch configuration-related installation errors before they occur.
+* *Advanced networking:* The {ai-full} supports IPv4/IPv6 dual stack networking, NMState-based static IP addressing, and an HTTP/S proxy.
+* *REST API:* The {ai-full} has a REST API, enabling automation.
+
+The {ai-full} is a great solution for:
+
+* *Pre-installation validation:* The {ai-full} validates the configuration before installation to ensure a high probability of success. Validation includes:
+  - Ensuring network connectivity
+  - Ensuring sufficient network bandwidth
+  - Ensuring connectivity to the registry
+  - Ensuring time synchronization between cluster nodes
+  - Verifying that the cluster nodes meet the minimum hardware requirements
+  - Validating the installation configuration parameters
+* *Multiple installation scenarios:* The {ai-full} supports numerous scenarios, including:
+  - Installing {product-title} on premises in a connected environment, including with an optional HTTP/S proxy.
+  - Installing Single Node OpenShift (SNO)
+  - Installing on vSphere
+  - Installing {VirtProductName} and {rh-storage} (formerly OpenShift Container Storage) from the web user interface
+  - Installing first clusters, demos, and proof of concept projects that don't have automation. Users can add automation to the {ai-full} through the API later
+* *Streamlined installation workflow:* The {ai-full} provides the installer as a service, which:
+  - Eliminates the need to install and run the installer locally
+  - Ensures the latest version of the installer up to the latest tested z-stream releases. Older versions remain available, if needed
+  - Enables building automation through the API without the need to run the installer locally
```

Comment:
These 3 high-level bullet points don't really follow from the title.  For example, "AI is a great solution for pre-installation validations"?  I recommend moving the first and third bullets (validations and streamlined installation workflow) up to where you discuss "Simplified deployment".  I recommend moving the second bullet up to the beginning of the file where you write "The {ai-full} supports the various deployment platforms with a focus on bare metal."

Reply:
Done